server/src/concepts/offer/routes/index.js

```diff
@@ -1,8 +1,14 @@
 import express from "express";
 import { createOffer } from "../controllers/index.js";
 import { getOffers } from "../controllers/index.js";
+import {
+  isAuthenticated,
+  isAdmin,
+} from "../../authorization/controllers/auth.js";
 
-router.post("/", createOffer);
+const router = express.Router();
+
+router.post("/", isAuthenticated, isAdmin, createOffer);
```

Comment:
```suggestion
router.post("/", isAuthenticated, createOffer);
```
To create offer is enough to be authenticated, to activate offer user must be an admin